app/services/middleware_topology_service.rb

```diff
@@ -11,11 +11,11 @@ class MiddlewareTopologyService < TopologyService
       :middleware_deployments,
       :middleware_datasources,
       :middleware_messagings,
-      :lives_on => [:host]
+      :lives_on => [:host, :container]
```

Comment:
are you sure that host and container should be listed here on the same level? does it work and display it ok?
the cross linking hierarchy is: eap->VM (aka here lives_on) -> host and eap->Container(aka lives_on), so container is on the same level as VM (lives_on), host is one level down (in most cases, where an entity doesn't live directly on a physical host, which is at the moment out of scope)